identity-service/src/notifications/processNotifications/favoriteNotification.js

```diff
@@ -0,0 +1,117 @@
+const models = require('../../models')
+const {
+  notificationTypes,
+  actionEntityTypes
+} = require('../constants')
+
+const getNotifType = (entityType) => {
```

Comment:
follow up, maybe we can have a single place where this is consolidated (we should have a new person handle this to spread out the knowledge)

Reply:
feel free to resolve no immediate action